frontend/src/scenes/persons/personsLogic.ts

```diff
@@ -257,6 +257,10 @@ export const personsLogic = kea<personsLogicType<PersonFilters, PersonLogicProps
         },
         navigateToTab: () => {
             if (props.syncWithUrl && router.values.location.pathname.indexOf('/person') > -1) {
+                if (values.activeTab !== PersonsTabType.HISTORY) {
+                    delete router.values.searchParams['page']
+                }
```

Comment:
NO. We do not mutate values.

Reply:
Ah, you spotted my deliberate mistake and pass the PR test 

![side-eye-monkey](https://user-images.githubusercontent.com/984817/160380746-7ccb8946-8489-444b-ad61-76f90cbcc400.png)